chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/product/UpdateOwnershipActionPermissions.java

```diff
@@ -29,14 +26,24 @@ public boolean hasAccess(Object obj) {
         } else if (getResourcePermissions().isUserRoleAdmin() || getResourcePermissions().isUserRoleOnc()) {
             return true;
         } else if (getResourcePermissions().isUserRoleAcbAdmin()) {
-            ProductDTO dto = (ProductDTO) obj;
-            if (getResourcePermissions().isDeveloperActive(dto.getDeveloperId())) {
-                return doesCurrentUserHaveAccessToAllOfDevelopersListings(dto.getDeveloperId());
-            } else {
+            try {
+                ProductDTO dto = (ProductDTO) obj;
+                // Get the original ProductDTO, since the one passed in has the "new" developer id
```

Comment:
The comment says "developer". Should it be "product"?

Reply:
The ProductDTO has the developerId, so I think this is correct.